Mobile.html

```diff
@@ -2,134 +2,192 @@
 <html lang="en">
   <head>
     <title>Little Prince</title>
-    <link rel="stylesheet" href="/style.css" >
+    <link rel="stylesheet" href="/style.css" />
     <script src="/script.js" defer></script>
   </head>
-<body>
-  <header>
-    <img class="logo" src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flogo.png?v=1605558587857">
-    <nav>
-        <a class="project" href="project.html"> Project </a>
-    </nav>
-  </header>   
+  <body>
+    <header>
+      <img
+        class="logo"
+        src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flogo.png?v=1605558587857" alt="logo"
+      />
+      <nav class="Project">
+        <a href="project.html"> Project </a>
+      </nav>
+    </header>
     <h1 class="home-title">Little Prince Space Travel</h1>
-    <img class="universe-background" src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Fspace.png?v=1605570976211">
-    <img class="little-prince" src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flittle%20prince.png?v=1604540994354">
+    <img
+      class="universe-background"
+      src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Fspace.png?v=1605570976211" alt="universe-background"
+    />
+    <img
+      class="little-prince"
+      src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flittle%20prince.png?v=1604540994354" alt="little-prince"
+    />
     <p class="introduction">
       The little Prince lost Rose and don't know where she disappeared. He is
       asking for your help and wants you to join him in this new adventure.
     </p>
-    <p class="introduction-2"> 
+    <p class="introduction-2">
       Start now and join the Little prince on his space journey!
     </p>
 
-    <div id="opentnContainer" style="opacity: 1; transition: all 0.5 ease 0s;">
-      <button id="openbtn" onclick="openRightMenu()" style="opacity: 1; transition: all 0.5 ease 0s;"> &lt; </button>
+    <p href="javascript:void(0)" class="openbtn" onclick="openNav()">&#8688;</p>
```

Comment:
Is a `p` tag the best tag to use here? I don't think `href` is valid on a `p` tag - an `a` might be a better choice here.